packages/ui-app/src/InputAddress/index.tsx

```diff
@@ -189,7 +187,7 @@ class InputAddress extends React.PureComponent<Props, State> {
       if (accountId) {
         if (!matches.find((item) => item.key === RECENT_KEY)) {
           matches.push(
-            createOptionHeader('Recent')
+            keyring.createOptionHeader('Recent')
```

Comment:
These I'm a bit sceptical about - well, not just here, overall :) It never felt like a good 100% fit, but was rather put there since it is the closest to a fit. So not convinced the option stuff (which is really UI-related, the rest of keyring is not). So maybe would rather keep this in it's own class? Any other suggestions?

Reply:
Yes, I've created a `KeyringOption` class for the options/ folder in ui-keyring